### PR TITLE
Prepare release notes for 1.2.3

### PR DESCRIPTION
**Pull request by serras**
Once the release is done, and the website updated, then the names of the libraries should link to the corresponding docs.

### CODE REVIEW

**Review by nomisRev — APPROVED** [review]
Looks great @serras! Thank you 😘

File: content/blog/2024-01-31-arrow-1-2-2.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Arrow provides building blocks relevant for many projects using Kotlin.
+A large part of our community is doing frontend work, and during the latest months
+the team has been trying to understand their needs, in order to make Arrow
+a relevant tool in that space.
+
+From that journey we have put together a new documentation page highlighting
+different ways in which Arrow may be useful in your Compose application.
+There is also a new `arrow-optics-compose` module that includes utilities to
+work with immutable data inside a `MutableState` or `MutableStateFlow`.
+
```
[the remark this answers is not in the document]
Comment (serras) [reply content/blog/2024-01-31-arrow-1-2-2.md:24]: Done!

**Comment by serras** [comment]
@nomisRev I've updated the release notes. This version is complete 🔥 !

**Comment by serras** [comment]
Did 1.2.2 occur at all? I only see 1.2.3 in these release notes...

**Comment by nomisRev** [comment]
> Did 1.2.2 occur at all? I only see 1.2.3 in these release notes...

? Not sure what you mean

**Comment by serras** [comment]
Sorry, it was intended to be a joke, as if somebody passed by and suddenly saw my commit changing from 1.2.2 to 1.2.3

**Pull request merged**